livekit-agents/livekit/agents/tts/tts.py

```diff
@@ -18,6 +18,7 @@ class TTSMetrics(TypedDict):
     duration: float
     audio_duration: float
     cancelled: bool
+    num_characters: int
```

Comment:
nit, following the same `<resource>_<unit>` convention

`character_count: int`